oss-cloud-backend/serverless.yml

```diff
@@ -14,3 +15,11 @@ functions:
       - http:
           path: hello
           method: get
+
+  getContributorPulls:
+    handler: handler.getPulls
+    events: # http event for testing, will probably change later
+      - http:
```

Comment:
This should be triggered periodically. I guess this is for now for testing, but please don't forget to add scheduled event.

Reply:
Added a TODO statement.